### PR TITLE
Slider , installation , colors page

### PR DESCRIPTION
**Pull request by dipen2000**
I created the slider component, installation, and colors page in my component library.

added favicon at last too.

So please review my PR and give your feedback.

### CODE REVIEW

File: CSS/colors.css [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
-    --test-color : rebeccapurple;
-    --light-test-color : #66339923;
+    --test-color : #663399;
+    --light-test-color : rgba(102, 51, 153, 0.356);
+    --status-online : #33ff00;
+    --status-offline : #939792;
+    --status-dnd : #F44336;
+    --danger-hover : #d31609;
+    --light-status-dnd : #f73e3e6b;
+    --info-alert-background : #a5f3fc;
```
Comment (dev-ashishsethi) [inline CSS/colors.css:13]: make variable name in capital

File: CSS/documentation.css [hunk 2]
```diff
@@ -68,23 +69,53 @@ main{
...
     grid-area: main-content;
 }
 
+.content-main-section h1{
+    font-size: 4rem;
+    font-family: var(--component-heading-font);
+}
+
+.content-main-section h2{
+    /* font-size: 4rem; */
```
Comment (dev-ashishsethi) [inline CSS/documentation.css]: please avoid commenting code

File: CSS/documentation.css [hunk 3]
```diff
@@ -159,8 +212,86 @@ main{
...
+
+.my-style:hover{
+    background-color: var(--rating-icon-hover);
+}
+
+.color-container{
+    width : 13rem;
+    height : 13rem;
+    /* border : 3px solid red; */
+    border-radius: 0.8rem;
```
Comment (dev-ashishsethi) [inline CSS/documentation.css:236]: If size is less than 1 rem then use pixels

**Review by dev-ashishsethi — APPROVED** [review]
there are some best practices mentioned please follow them

File: components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+.btn-rounded{
+    border-radius: 30px;
+}
+
+.btn-floating{
+    height : 5rem;
+    width : 5rem;
+    padding : 10px;
```
Comment (Ananyamadhu08) [inline components/button/button.css]: use rem instead of px

File: components/slider/slider.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
+.slider-rate{
+    text-align:center;
+    font-size : 2.5rem;
+    font-family: var(--component-heading-font);
+}
+
+.slider-container{
+    width : 30rem;
+    /* height : 6.15rem; */
+    /* border : 3px solid green; */
```
Comment (Ananyamadhu08) [inline components/slider/slider.css]: can delete commented code

**Review by Ananyamadhu08 — APPROVED** [review]
Use rem instead of px, other than that looks good to go 🚀

**Pull request merged**